types/open-location-code/open-location-code-tests.ts

```diff
@@ -0,0 +1,29 @@
+import OpenLocationCode from "open-location-code";
```

Comment:
Use the commonjs syntax `import OpenLocationCode = require('open-location-code')` to match the commonjs `export=` syntax.

Consumers of the package who want to use `import OpenLocationCode from 'open-location-code'` will need to add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.

Reply:
I wasn't aware of the testing conventions, so I was using ESM-style imports. I'll switch to using CommonJS-style imports for testing.